mrSerious/StackOverflow-lite
# #160185465 API documentation

#### What does this PR do?
Have documentation for API endpoints
#### Description of Task to be completed?
Have the following endpoint that documents the API application: `/api-docs/`
#### How should this be manually tested?
- Open your browser
- Type in the route
- You should be presented with a view of the API documentation
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#160185465](https://www.pivotaltracker.com/n/projects/2189147/stories/160185465)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
